src/lib/molecules/hero-slider.svelte

```diff
@@ -1,159 +1,157 @@
-    <script>
-        export let data; 
-
-        // import { HeroSlide } from '$lib/index.js';
-        import { onDestroy, onMount } from 'svelte';
-
-        // @ts-ignore
-        let intervalId;
-
-        onMount(() => {
-            const carrouselContainer = document.querySelector('.hero-slider');
-            const carrousel = document.querySelector('.hero-img-list');
-
-            const interval = 8000;
-
-            intervalId = setInterval(() => {
-                // @ts-ignore
-                const currentScroll = carrousel.scrollLeft;
-                // @ts-ignore
-                const itemWidth = carrouselContainer.clientWidth;
-                // @ts-ignore
-                const totalWidth = carrousel.scrollWidth;
-
-                const nextScroll = (currentScroll + itemWidth) % totalWidth;
-
-                // @ts-ignore
-                carrousel.scrollTo({
-                    left: nextScroll,
-                    behavior: 'smooth'
-                });
-            }, interval);
-        });
-
-        onDestroy(() => {
-            // @ts-ignore
-            clearInterval(intervalId);
-        });
-
-        onDestroy(() => {
-            // @ts-ignore
-            clearInterval(intervalId);
-        });
-    </script>
+<script>
+	export let data;
+
+	// import { HeroSlide } from '$lib/index.js';
+	import { onDestroy, onMount } from 'svelte';
+
+	// @ts-ignore
+	let intervalId;
+
+	onMount(() => {
+		const carrouselContainer = document.querySelector('.hero-slider');
+		const carrousel = document.querySelector('.hero-img-list');
+
+		const interval = 8000;
+
+		intervalId = setInterval(() => {
+			// @ts-ignore
+			const currentScroll = carrousel.scrollLeft;
+			// @ts-ignore
+			const itemWidth = carrouselContainer.clientWidth;
+			// @ts-ignore
+			const totalWidth = carrousel.scrollWidth;
+
+			const nextScroll = (currentScroll + itemWidth) % totalWidth;
+
+			// @ts-ignore
+			carrousel.scrollTo({
+				left: nextScroll,
+				behavior: 'smooth'
+			});
+		}, interval);
+	});
+
+	onDestroy(() => {
+		// @ts-ignore
+		clearInterval(intervalId);
+	});
+
+	onDestroy(() => {
+		// @ts-ignore
+		clearInterval(intervalId);
+	});
+</script>
 
 <section>
-    <section class="hero-slider">
-        <div class="overlay"></div>
-
-        <header>
-            <h1> Samen <span>Groener</span> Leven</h1>
-            <h2>Inspireer & Deel je Groene Passie!</h2>
-        </header>
-
-        <ul class="hero-img-list">   
-                {#each data.herosSlider as hero}
-                    <li>
-                        <picture>
-                            <img src="{hero.sliderPicture.url}" alt="{hero.sliderPicture.altText}" />
-                        </picture>
-                    </li>
-                {/each}
-        </ul>
-    </section>
+	<section class="hero-slider">
```

Comment:
Deze section is direct genest in een parent section. Dat is semantisch gezien niet correct.